portal-web/test/functional/com/liferay/portalweb/tests/mytests/documentsAM.testcase

```diff
@@ -0,0 +1,105 @@
+@component-name = "portal-document-management"
+definition {
+
+	property custom.properties = "default.layout.template.id=1_column${line.separator}dl.actions.visible=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Document Management";
+
+	var pageName = "Documents and Media Page";
+	var portletName = "Documents and Media";
+
+	setUp {
+		task ("Create portal instance and login") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Create page of DM") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Documents and Media Page");
+		}
+
+		task ("Add a widget for DM type") {
+			JSONLayout.addWidgetToPublicLayout(
+				column = "1",
+				groupName = "Guest",
+				layoutName = "Documents and Media Page",
+				widgetName = "Documents and Media");
+		}
+
+		task ("Go to the page DM") {
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Documents and Media");
+		}
+
+		task ("Add a image in the DM page") {
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentTitle = "imagem 1",
+				groupName = "Guest",
+				mimeType = "image/jpeg",
+				sourceFileName = "high_resolution_photo.jpg");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			DMDocument.tearDownCP();
+
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "Documents and Media Page");
+		}
+	}
+
+	@description = "Editing a DM"
+	@priority = "5"
+	test CanDeleteDM {
+		task ("Create page of DM") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Documents and Media Page");
+		}
+
+		task ("Add a widget for DM type") {
+			JSONLayout.addWidgetToPublicLayout(
+				column = "1",
+				groupName = "Guest",
+				layoutName = "Documents and Media Page",
+				widgetName = "Documents and Media");
+		}
+
+		task ("Go to the page DM") {
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Documents and Media");
+		}
+
+		task ("Add a image in the DM page") {
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentTitle = "imagem 2",
+				groupName = "Guest",
+				mimeType = "image/jpeg",
+				sourceFileName = "Document_1.jpg");
+		}
```

Comment:
Oi @JhossephG, você já colocou isso no setUp, você acha que precisa ficar aqui também?

Reply:
Oi, @ViniLopes20 , a parte a ser removida seria essa ai abaixo, correto?

task ("Create page of DM") {
            JSONLayout.addPublicLayout(
                groupName = "Guest",
                layoutName = "Documents and Media Page");
        }
        task ("Add a widget for DM type") {
            JSONLayout.addWidgetToPublicLayout(
                column = "1",
                groupName = "Guest",
                layoutName = "Documents and Media Page",
                widgetName = "Documents and Media");
        }
        task ("Go to the page DM") {
            ProductMenu.gotoPortlet(
                category = "Content & Data",
                portlet = "Documents and Media");
        }

Reply:
Isso